README.md

```diff
@@ -93,6 +93,27 @@ try {
 }
 ```
 
+## Basic usage of `exads-api-client-php` client with API token authentication
+
+
+```php
+<?php
+
+require_once 'vendor/autoload.php';
+
+try {
+    $client = new Exads\Client('https://api.exads.com/url/');
+    $apiToken = $client->login->getToken('APItoken');
```

Comment:
Sure.
I changed the name of the $apiToken variable to $sessionToken as the API (doctumetntation) tries to use the "session token" and "api token" terms